plugin/trino-delta-lake/pom.xml

```diff
@@ -249,12 +249,24 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
```

Comment:
If we can convert `TestDeltaLakeAlluxioCacheFileOperations` to construct the connector with properties and use tracing (per my other comment), then we should be able to remove these runtime dependencies.